src/modules/app-error/app-error.ts

```diff
@@ -0,0 +1,15 @@
+export default class AppError extends Error {
+  code: string;
+
+  httpStatusCode: number;
+
+  toString(): string {
+    return `${this.code}: ${this.message}`;
+  }
+
+  toJson(): unknown {
```

Comment:
The default implementation should be to loop through all properties available on the instance and render it as a json